avocado/utils/configure_network.py

```diff
@@ -189,6 +187,16 @@ def get_peer_interface(self, peer_ip):
         else:
             return peer_interface
 
+    def is_interface_link_up_peer(self, peer_interface):
+        """
+        check if peer interface link is up
```

Comment:
Also, I would like to suggest a better (multi-line) docstring here:

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. 

https://www.python.org/dev/peps/pep-0257/